_posts/2016-04-22-welcome-to-beyond-scheme-blog.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  "Welcome to Beyond Scheme blog!"
+date:   2016-04-22 16:07:07 +0100
+author: "Artur Trzop"
+categories: news
```

Comment:
This is prefix in the post url. For instance:
`http://127.0.0.1:4000/news/2016/04/22/welcome-to-beyond-scheme-blog.html`

see news as prefix before date /news/2016/04/22/welcome-to-beyond-scheme-blog.html